tests/integration/components/freestyle/usage/object-test.js

```diff
@@ -30,7 +30,7 @@ module('Integration | Component | freestyle/usage/object', function (hooks) {
     await render(hbs`<Freestyle::Usage::Object
       @name="Collectibles"
       @value={{this.value}}
-      @jsonCollapseDepth={{1}}
+      @jsonCollapseDepth={{3}}
```

Comment:
Aside from the different look, the way both libraries handle the initial open state is different as well. It seems `ember-json-viewer` works differently when arrays are included. When that's not the case they show the same level of nesting initially.